SimplyPluralApi.yml

```diff
@@ -1750,43 +1837,67 @@ paths:
       responses:
         "200":
           description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "./SimplyPluralApiBodies.yml#/components/schemas/chatCategoryContainer"
         "404":
           description: NOT FOUND
 
-  /v1/chat/channel/:id:
+  /v1/chat/channel:
```

Comment:
Don't create seperate endpoints for one and the same endpoint with an optional id.